src/radical/entk/constants.py

```diff
@@ -3,4 +3,4 @@
 __license__   = 'MIT'
 
 
-NAME_MESSAGE = "Valid object names can contain letters, numbers and '.'. Any other character is not allowed"
+NAME_MESSAGE = "Valid object names can contain letters, numbers and '.'. Any other character will not be allowed in version 1.5.10"
```

Comment:
"Valid object names can contain letters, numbers and the symbol '.'. Any other character will not be allowed in version >= 1.5.10"?